modules/gapi/include/opencv2/gapi/gtype_traits.hpp

```diff
@@ -26,7 +26,10 @@ namespace detail
     // a double dispatch
     enum class ArgKind: int
     {
-        OPAQUE,       // Unknown, generic, opaque-to-GAPI data type - STATIC
+        GOPAQUE,      // Unknown, generic, opaque-to-GAPI data type - STATIC
```

Comment:
`OPAQUE` is not `GOPAQUE`.

`OPAQUE` is every value which doesn't have a G-type for it.

`GOPAQUE` should be a `GOpaque<T>` thing for "manageable" user-defined types  in graph I/O. We still have that in our backlog :)

Reply:
Proposed: `OPAQUE_VAL` or `OPAQUE_VALUE` (see what I had to do with `GModel::CONST` previously)

Reply:
agree, changed to `OPAQUE_VAL` to align with `Storage::CONST_VAL`